spec/emitter_spec.rb

```diff
@@ -590,35 +588,33 @@
   describe "scrubbing" do
     it "scrubs based on key name" do
       configuration.scrubber = ->(key, value) do
-        key =~ /password/ ? "[scrubbed]" : value
+        /password/.match?(key) ? "[scrubbed]" : value
       end
 
-      subject.log hello: :world, my_password: "this should be scrubbed"
+      subject.log(hello: :world, my_password: "this should be scrubbed")
       expect(output).to eq("hello=world my-password=[scrubbed]\n")
     end
 
     it "scrubs based on value" do
       configuration.scrubber = ->(key, value) do
-        value =~ /secret/ ? "[scrubbed]" : value
+        /secret/.match?(value) ? "[scrubbed]" : value
       end
 
-      subject.log \
-        hello: :world,
-        my_password: "this secret schould be scrubbed",
-        other_field: -> { "also secret, should be scrubbed" }
+      subject.log(hello: :world,
+                  my_password: "this secret schould be scrubbed",
+                  other_field: -> { "also secret, should be scrubbed" })
 
       expect(output).to eq("hello=world my-password=[scrubbed] other-field=[scrubbed]\n")
     end
 
     it "allows to omit tokens when value is nil" do
       configuration.scrubber = ->(key, value) do
-        value =~ /secret/ ? nil : value
+        /secret/.match?(value) ? nil : value
       end
 
-      subject.log \
-        hello: :world,
-        my_password: "this secret schould be scrubbed",
-        other_field: -> { "also secret, should be scrubbed" }
+      subject.log(hello: :world,
+                  my_password: "this secret schould be scrubbed",
+                  other_field: -> { "also secret, should be scrubbed" })
```

Comment:
Omg, does standardrb really suggests this awful syntax?

Reply:
Not exactly. This had some manual intervention. The automated formatting actually left the `\`, even with the parens. Hopefully the forthcoming `rubyfmt` will have a better answer there.

Anyhow, I removed the then redundant `\` and pulled the lines up. At that point the auto-formatting resulted in what we see here. If we push the first line back down, the auto-formatting will result in this:

```ruby
subject.log(
  hello: :world,
  my_password: "this secret schould be scrubbed",
  other_field: -> { "also secret, should be scrubbed" }
)```

Which I can totally live with. Maybe we should go that route so it's less churn?

Reply:
**UPDATE**: I rebased in a change to keep the original formatting, but with parens. 😄

Reply:
I like it way more, thanks!